### ISSUE TITLE
about rev_rec_plus

### ISSUE THREAD

**Issue by ChenCherry** [Kali-Hac/Locality-Awareness-SGE#2]
If I set Model = 'rev_rec_plus'in trai.py (line 64), the results is not good as  Model = 'rev_rec'. Do I need to modify other parameters, like pre_task = 'rev_rec' in trai.py (line 65)？ Thanks a lot.

**Comment by Kali-Hac** [comment]
Main Reasons:
1. In our paper, all experiments are repeated for multiple times on a specific setting (a different dataset or model setting) and the average performance is reported.  
2. Since we do NOT explicitly select or set a same random seed for our experiments, the results in every training may have some slight randomness, i.e., the parameters in neural networks are randomly initialized at the beginning.  Despite this fact, according to the extensive experimental results, the setting 'rev_rec_plus' is better in average than 'rev_rec' on three datasets, namely BIWI, KS20, and KGBD datasets. 

Solutions:
1. Considering that results of 'rev_rec_plus' is quite close to 'rev_rec' on BIWI and IAS-Lab datasets, you can simply run the same setting for several times to compare their average performance.
2. You can also change the hyper-parameters provided in the source codes, such as temperature (t), epochs, rnn_size to get further potential improvement (note that the default parameters are not necessarily the optimal),
3. You can directly download the best models provided in the repository. They can reproduce the same experimental results in our paper.

Thank you for your attention.
